components/postgresql/actions/find-row-custom-query/find-row-custom-query.mjs

```diff
@@ -0,0 +1,46 @@
+import postgresql from "../../postgresql.app.mjs";
+
+export default {
```

Comment:
This particular component is kind of confusing becase the name suggests it's only for finding rows with a custom query but at the same time the query is too open to even run an insert/update/delete as well, so I think this one could easily be replaced by `execute-custom-query` don´t you think so?

Reply:
I agree. `find-row-custom query` does the same thing as `execute-custom-query` but only returns the first result. I removed this component.